tests/components/test_question_answerer.py

```diff
@@ -274,53 +276,45 @@ def test_advanced_search_validation(answerer):
 @pytest.mark.bert
 @pytest.mark.es7
 def test_embedder_search_bert(food_ordering_with_bert):
-    BERT_EMBEDDING_LEN = 768
     res = food_ordering_with_bert.get(
         index="menu_items_bert", query_type="embedder", name="pasta with tomato sauce"
     )
     assert len(res) > 0
```

Comment:
why do we remove these assertions?

Reply:
I removed the embedding from the item returned from the question answerer, since it seems cleaner and we don't return similar metadata for text features